pages/portfolio.html

```diff
@@ -1,12 +1,92 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="../styles/style.css" rel="stylesheet" />
+  <link href="../styles/portfolio.css" rel="stylesheet" />
+
 </head>
+
 <body>
-  
+  <div class="portfolio-content2">
+
+
+
+    <nav class="nav-container">
+
+      <ul>
+        <li class="nav-item active">
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About me</a>
+        </li>
+        <li>
+          <a href="portfolio.html"  class="active">Portfolio</a>
+        </li>
+
+      </ul>
+
+    </nav>
+
+
+    <div class="portfolio-content">
+      <div class="prj">
+        <h2>Projects</h2>
+      </div>
+      <div class="projects">
+        <div class="project-1">
```

Comment:
Having class names like `project-1` and `project-2` could instead by accomplished by using something like the [nth-of-type selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type) which would DRY up your code.